cmd/provisioner-localpv/app/helper_hostpath.go

```diff
@@ -92,6 +104,63 @@ func (pOpts *HelperPodOptions) validate() error {
 	return nil
 }
 
+//validateLimits check that the limits to setup qouta are valid
+func (pOpts *HelperPodOptions) validateLimits() error {
+	if pOpts.bsoft == "0k" &&
+		pOpts.bhard == "0k" {
+		return errors.Errorf("both limits cannot be 0")
```

Comment:
Return errors in detailed format
```suggestion

klog.Infof("Setting xfs quota for volume %v failed: %v", name, err)
				alertlog.Logger.Errorw("",
					"eventcode", "local.pv.quota.failure",
					"msg", "Failed to set xfs quota limits for Local PV",
					"rname", opts.PVName,
					"reason", "xfs quota validation failed",
					"storagetype", stgType,
				)
				return nil, err
```

Reply:
`PVName` is not available in `helper_hostpath.go` so I should skip it , right?

Reply:
and `stgType` is also not available

Reply:
Okay. Then leave it. Since in provisioner_hostpath.go the logs are properly errored out.